src/components/custom/_button.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import styled from 'styled-components'
+import { TString } from 'types/generics'
+
+type TProps = {
+    label: TString
+    onClick?: VoidFunction
+    primary?: boolean
+    secondary?: boolean
+    tertiary?: boolean
+    outline?: boolean
+    hero?: boolean
+}
+
+const Label = styled.span<Omit<TProps, 'label' | 'onClick'>>`
```

Comment:
```suggestion

type TLabelProps = Omit<TProps, 'label' | 'onClick'>

type TContainerProps = Omit<TProps, 'label'>

const Label = styled.span<TLabelProps>`...`

const Container = styled.div<TContainerProps>`...`
```
@mitra-deriv Just a suggestion for better readability 🙇🏻

Reply:
@farzin Thank you for the suggestions I have a Todo which I need to address them later will apply your comments in that branch, Once done in that branch will resolve these comments. 🌹 🙏

Reply:
Solved in this PR #4114